w3af/core/controllers/core_helpers/status.py

```diff
@@ -166,13 +166,13 @@ def get_rpm(self):
                  start of the scan.
         """
         if self._start_time_epoch is None:
-            raise RuntimeError('Can NOT call get_run_time before start().')
-        
+            return None
```

Comment:
This is called from `/scans/<scan_id>/status`, via `w3af.core.status.get_status_as_dict()`
The dict looks like this:
```
 data = {                                                        
         'status': self.get_simplified_status(),                 
         'is_paused': self.is_paused(),                          
         'is_running': self.is_running(),                        
                                                                 
         'active_plugin':                                        
             {'crawl': self.get_running_plugin('crawl'),         
              'audit': self.get_running_plugin('audit')}         
         ,                                                       
                                                                 
         'current_request':                                      
             {'crawl': crawl_fuzzable_request,                   
              'audit': audit_fuzzable_request},                  
                                                                 
         'queues':                                               
             {'crawl':                                           
                 {'input_speed': self.get_crawl_input_speed(),   
                  'output_speed': self.get_crawl_output_speed(), 
                  'length': self.get_crawl_qsize()},             
              'audit':                                           
                 {'input_speed': self.get_audit_input_speed(),   
                  'output_speed': self.get_audit_output_speed(), 
                  'length': self.get_audit_qsize()}              
             },                                                  
                                                                 
         'eta':                                                  
             {'crawl': self.get_crawl_eta(),                     
              'audit': self.get_audit_eta()},                    
                                                                 
         'rpm': self.get_rpm(),                                  
         }                                                       
```

Of those, only `get_rpm()` was throwing an exception if called before the scan was started - the rest return 'None', though functions such as `get_crawl_eta()` are equally meaningless if the scan has never run.

It may be more correct to patch the other functions so that they throw an exception, and simply handle the exceptions in `/scans/<scan_id>/status` instead?